raiden-ts/docs-source/private-chain.md

```diff
@@ -72,83 +88,101 @@ import privateServicesDeployment from '@/deployment/deployment_services_private_
+  } as unknown as ContractsInfo;
 
-    const raiden = await Raiden.create(
-        provider,
-        0,
-        window.localStorage,
-        contractsInfo
-    );
+  const raiden = await Raiden.create(provider, 0, window.localStorage, contractsInfo);
 }
 ```
 
 Calling the `initialization()` function should enable the SDK to manage channels and send payments in the development environment, using the previously deployed token.
 
 ### Changing the dApp to work on the private setup
+
 Out of the box the dApp supports the following public test networks `Görli`, `Rinkeby` and `Ropsten`.
 
 To run the dApp on any other network setup you need to deploy the smart contracts and change the dApp to use this setup.
 
 If you require help running the dApp, you can follow the [instructions].
 
-As soon as you have the dApp running you can locate the [`createRaiden`] function and change it accordingly. 
+As soon as you have the dApp running you can locate the [`createRaiden`] function and change it accordingly.
 
 First, you need to import the deployment information, as with the SDK setup.
 
 ```typescript
 import privateDeployment from '@/deployment/deployment_private_net.json';
 import privateServicesDeployment from '@/deployment/deployment_services_private_net.json';
-import { ContractsInfo } from 'raiden-ts';
+import type { ContractsInfo } from 'raiden-ts';
 ```
 
-Then you need to construct the `contractsInfo` object. 
+Then you need to construct the `contractsInfo` object.
 
 The [`createRaiden`] function should look like the example below:
 
 ```typescript
   private static async createRaiden(
-    provider: any,
-    account: string | number = 0
+    provider: providers.JsonRpcProvider | string,
+    privateKeyOrProviderAccountIndex: string | number = 0,
+    stateBackup?: string,
+    subkey?: true,
   ): Promise<Raiden> {
-    const contractsInfo = ({
+    const contractsInfo = {
       ...privateDeployment.contracts,
-      ...privateServicesDeployment.contracts
-    } as unknown) as ContractsInfo;
-    try {
+      ...privateServicesDeployment.contracts,
+    } as unknown as ContractsInfo;
+..
+..
+..
+  try {
+      // const contracts = await ConfigProvider.contracts();
+
       return await Raiden.create(
         provider,
-        account,
-        window.localStorage,
+        privateKeyOrProviderAccountIndex,
+        storageOpts,
         contractsInfo,
-        { pfsSafetyMargin: 1.1 }
-      );
-    } catch (e) {
-      throw new RaidenInitializationFailed(e);
-    }
-  }
+        {
+          pfsSafetyMargin: 1.1,
+          ...(process.env.VUE_APP_PFS
+          ..
+          ..
+```
+
+Comment out `const contracts = await ConfigProvider.contracts();` and add `contractsInfo` just below the `storageOpts` variable.
+
+Finally with these changes you need to run
+
+```bash
+yarn workspace raiden-dapp serve
+```
+
+You might face issues with linting for that you will need to go in the `raiden-dapp` subfolder and run the following:
```

Comment:
Why might this happen?

Reply:
i have told the user to run the command 
`yarn workspace raiden-dapp serve`
In this what happens on few occassions is that we might get lint errors but the `serve` command considers it as actual errors and the dapp is not served. So if we just fix linting and then serve then it works. Dont know why this peculiar behavior but it does happen on and off.